liblangutil/ErrorReporter.cpp

```diff
@@ -94,6 +94,16 @@ bool ErrorReporter::checkForExcessiveErrors(Error::Type _type)
 		if (m_warningCount >= c_maxWarningsAllowed)
 			return true;
 	}
+	else if (_type == Error::Type::Info)
+	{
+		m_infoCount++;
+
+		if (m_infoCount == c_maxInfosAllowed)
+			m_errorList.push_back(make_shared<Error>(2833_error, Error::Type::Info, "There are more than 256 infos. Ignoring the rest."));
```

Comment:
Interesting meta question: should these "ignoring the rest" messages be of the appropriate error type? :)

I think that makes the most sense.

Reply:
If too many warnings is also an error, then I also think it should be an error here.

Reply:
It seems they are set as error, warning, info for the appropriate severity, I think that makes sense. So no change required here.